--- origsrc/gmime-2.6.0/configure.ac
+++ src/gmime-2.6.0/configure.ac
@@ -579,8 +579,8 @@
 if test "x$LIBNSL" != "x"; then
 	EXTRA_LIBS="$EXTRA_LIBS $LIBNSL"
 fi
-if test "x$LIBICONV" != "x"; then
-	EXTRA_LIBS="$EXTRA_LIBS $LIBICONV"
+if test "x$LTLIBICONV" != "x"; then
+	EXTRA_LIBS="$EXTRA_LIBS $LTLIBICONV"
 fi
 if test "x$GPGME_PTHREAD_LIBS" != "x"; then
 	EXTRA_LIBS="$EXTRA_LIBS $GPGME_PTHREAD_LIBS"
